src/utils/contentTypes.ts

```diff
@@ -63,3 +63,18 @@ export const fileExtensionFromResourceEncoding = (encodingType?: string) => {
 
   return extension;
 };
+
+export const getCorrectFileExtension = (
```

Comment:
by `correct` i mean: 
since we have different source to get the extension than we should get the correct one, because if we take from one source than it might be wrong (coz, we have another source for extracting that extension if it exist)

Reply:
In that case, I think you want to call the function `getNormalizedExtension()` since it takes care of considering the different sources and retrieving the extension from it.

If you have created a function or any utility you always want it to be "correct". The meaning of "correct" itself is determined by the implementation details of that function/utility and the tests you wrote, and hence  correctness is always implied.

I guess it'll be clearer if I give a counter example, would you ever create a function called "getWrongExtesion()" (except maybe in a test, when creating a mock)? Probably not.  Therefore the word "correct" is not useful, and potentially confusing in a name.